go.mod

```diff
@@ -1,6 +1,6 @@
 module istio.io/istio
 
-go 1.22
+go 1.22.0
```

Comment:
Interesting that the .0 was added. Maybe to not allow the use of beta to release candidate directives in dependencies.

Reply:
Yeah I am not sure why k8s did it, but them doing it forces us to do it so... not much choice :shrug: